package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "ccdb5-ui",
+  "version": "5.0.0",
+  "private": false,
+  "description": "Consumer Complaint Database 5.0 UI",
+  "homepage": "http://www.consumerfinance.gov/",
+  "author": {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/cfpb/ccdb5-ui.git"
+  },
+  "license": "CC0-1.0",
+  "dependencies": {
+    "capital-framework": "^4.3.2",
+    "history": "^4.6.1",
+    "normalize-css": "^2.3.1",
```

Comment:
If we're not supporting IE until 9, I think you can use a more recent version of Normalize.